docs/_interface-customization.md

```diff
@@ -2,13 +2,13 @@
 title: Modifying the LibreTime interface
 ---
 
-The LibreTime administration interface, as a web application, is fully customizable using the same methods that you might use to modify a website. For instance, you may wish to increase certain font sizes or change the colours in the LibreTime interface to better suit staff users with impaired vision. To do so, open one of the CSS files in the _/public/css/_ directory under the LibreTime _DocumentRoot_ directory in an editor such as **nano**:
+The LibreTime administration interface, as a web application, is fully customizable using the same methods that you might use to modify a website. For instance, you may wish to increase certain font sizes or change the colours in the LibreTime interface to better suit staff users with impaired vision. To do so, open one of the CSS files in the `/public/css/` directory under the LibreTime _DocumentRoot_ directory in an editor such as `nano`:
```

Comment:
In this line for example, we have **nano**, I would not put any editor's name in the first place, but I would rather put this one between back tick as adding a rule in the accepted names.

This is mainly because nano has nothing to do with our project, but Icecast totally do.

But this is has more something to do  more because

Reply:
Yup, I'm happy with that